src/test/denialofservice_tests.cpp

```diff
@@ -8,13 +8,15 @@
 #include <chainparams.h>
 #include <net.h>
 #include <net_processing.h>
+#include <netmessagemaker.h>
```

Comment:
nit in the second commit: Can be removed?

Reply:
Done.